packages/scss/src/theming/components/_checkbox.theme.scss

```diff
@@ -0,0 +1,4 @@
+$checkbox: (
+  color: _color("secondary"),
```

Comment:
you just have to define `default-palette: _theme("palettes.secondary")` here and you're good